### DISCUSSION TITLE
How to Install AutoHotkey.py

### DISCUSSION THREAD

**Discussion by Gaby281** [Perlence/AutoHotkey.py#28] [Q&A] [answered]
When I use "py -m pip install --user autohotkey.py" I get this error: The term 'py' is not recognized as the name of a cmdlet, function, script file, or operable program.

**Comment by Perlence** [comment, marked as answer]
Hey @Gaby281, I suggest installing the latest Python version with the [official Windows installer](https://www.python.org/downloads/windows/). It will also install `py`, the [Python launcher](https://docs.python.org/3/using/windows.html#launcher), which you can use to install AutoHotkey.py.
